int32 id
semap_msgs/Polygon2DModel model
---
int32 id

